pipelines/sync.go

```diff
@@ -0,0 +1,56 @@
+package pipelines
+
+import (
+	syncPipeRes "github.com/jfrog/jfrog-cli-core/v2/pipelines/commands"
+	clientlog "github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/urfave/cli"
+)
+
+// syncPipelineResources sync pipelines resource
+func syncPipelineResources(c *cli.Context) error {
+	branch := c.String("branch")
+	repository := c.String("repository")
+	serverID := c.String("server-id")
+	clientlog.Info("🐸🐸🐸 Triggering pipeline sync on repository ", repository, "branch", branch)
```

Comment:
We have 2 options:
1. Remove the 3 frogs
2. Add the 3 frogs to *all* commands in the JFrog CLI :)
```suggestion
	clientlog.Info("Triggering pipeline sync on repository", repository, "branch", branch)
```